app/docs/0.12.x/loadbalancing.md

```diff
@@ -87,10 +76,32 @@ The DNS resolver will start resolving the following record types in order:
   3. A record
   4. CNAME record
 
-So if the hostname you use has both SRV entries and A entries, it will start
-with SRV. If you want A records to be used, you must remove the SRV records from
-the DNS server. If you only have A records, then the SRV lookup will fail and
-it will fallback on an A query, etc.
+This order is configurable through the [`dns_order` configuration property][dns-order-config].
+
+[Back to TOC](#table-of-contents)
+
+#### **DNS caveats**
+
+- Whenever the DNS record is refreshed a list is generated to handle the
+weighting properly. Try to keep the weights as multiples of each other to keep
+the algorithm performant, e.g., 2 weights of 17 and 31 would result in a structure 
+with 527 entries, whereas weights 16 and 32 (or their smallest relative 
+counterparts 1 and 2) would result in a structure with merely 3 entries,
+especially with a very small (or even 0) `ttl` value.
+
+- Some nameservers do not return all entries (due to UDP packet size) in those
+cases (for example Consul returns a maximum of 3) Kong will only use the ones
+provided and hence be limited. To mitigate this use another nameserver, use IP
```

Comment:
"...a given Kong node will only use the few upstream service instances provided by the nameserver. In this scenario, it is possible that the pool of upstream instances will be loaded inconsistently, because Kong is effectively unaware of some of the instances, due to the limited information provided by the nameserver."